plugins/module_utils/network/ios/facts/service/service.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+
+__metaclass__ = type
+
+"""
+The ios_service fact class
+It is in this file the configuration is collected from the device
+for a given resource, parsed, and the facts tree is populated
+based on the configuration.
+"""
+
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common import utils
+
+from ansible_collections.cisco.ios.plugins.module_utils.network.ios.argspec.service.service import (
+    ServiceArgs,
+)
+from ansible_collections.cisco.ios.plugins.module_utils.network.ios.rm_templates.service import (
+    ServiceTemplate,
+)
+
+
+class ServiceFacts(object):
+    """The ios service facts class"""
+
+    def __init__(self, module, subspec="config", options="options"):
+        self._module = module
+        self.argument_spec = ServiceArgs.argument_spec
+
+    def get_service_data(self, connection):
+        return connection.get("show running-config all | section ^service ")
+
+    def sort_list_dicts(self, objs):
```

Comment:
Sorting is not required to generate facts, a lot of our modules have it we are planning a cleanup activity.
Regards